test/conf/config.json

```diff
@@ -1,5 +1,4 @@
 {
-  "DB_NAME": "test_bartender",
```

Comment:
It's not obvious to me where this file gets used, if at all.  If it were being used, I would think removing this line altogether would break something.  Let's just delete this file (and the directory it's in since it would then be empty) and if the integration tests break even worse than they're already broken, I'll decide what to do about that.